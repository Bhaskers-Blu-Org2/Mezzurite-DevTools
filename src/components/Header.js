import React, { Component } from 'react';

class Header extends Component {

    componentDidMount() {
        
    }

    componentWillUnMount() {

    }

    render() {
        return (
            <header className="Header">
                <img src='./icon.png'></img>
                <h1>Mezzurite Developer Tools</h1>
            </header>
        );
    }
}

export default Header;