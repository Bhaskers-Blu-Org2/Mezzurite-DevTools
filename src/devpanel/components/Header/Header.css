@media only screen {
  .header {
    align-items: center;
    background-color: var(--microsoft-gray-darkest);
    color: var(--microsoft-white);
    display: flex;
    justify-content: center;
    padding: 10px;
  }

  .header--text {
    font-size: 16px;
    margin: 0;
    padding: 0;
  }
}

@media only screen and (min-width: 540px) {
  .header {
    justify-content: start;
  }

  .header--text {
    font-size: 24px;
    text-align: start;
  }
}
