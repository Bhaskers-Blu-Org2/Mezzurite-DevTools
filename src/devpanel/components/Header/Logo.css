@media only screen {
  .logo-inner {
    fill: var(--microsoft-white);
  }

  .logo {
    background-color: var(--microsoft-orange);
    border-radius: 16px;
    margin-right: 8px;
    padding: 2px;
  }
}

@media only screen and (min-width: 540px) {
  .logo {
    padding: 4px;
  }
}
