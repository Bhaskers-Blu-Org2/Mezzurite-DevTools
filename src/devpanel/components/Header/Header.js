import React, { Component } from 'react';
import './Header.css';

class Header extends Component {
  componentDidMount () {

  }

  componentWillUnMount () {

  }

  render () {
    return (
      <header className='Header'>
        <h1>Mezzurite Developer Tools</h1>
      </header>
    );
  }
}

export default Header;
