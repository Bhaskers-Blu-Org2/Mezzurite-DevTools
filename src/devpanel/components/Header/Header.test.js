import Header from './Header';

describe('Header.js', () => {
  it('should render the header', () => {
    expect(Header()).toMatchSnapshot();
  });
});
