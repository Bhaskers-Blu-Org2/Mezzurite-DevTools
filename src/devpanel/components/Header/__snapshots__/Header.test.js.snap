// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.js should render the header 1`] = `
<header
  className="header"
>
  <h1
    className="header--text"
  >
    Mezzurite Developer Tools
  </h1>
</header>
`;
